reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+    properties:
+      description:
+        type: string
+        description: Reason for the discount.
+      value:
+        type: string
+        description: The value of the discount. If the type of the discount is `fixed_amount`, then this is a fixed dollar amount, if the type is `percentage`, then this is the percentage. 
+      type:
+        type: string
+        description: The type of the discount. Can be one of - `percentage` or `fixed_amount`. `Fixed_amount` is default value.
+        enum:
+        - percentage
+        - fixed_amount (default)
+
+      amount:
+        type: string
+        description: The applied amount of the discount.
+
+  customer:
+    type: object
+    description: Customer to be associated with the draft order; however only the `id` can be set in order to associate the customer to the draft order.  Setting the value to null removes the customer from the draft order.
+    properties:
+      accepts_marketing:
+        type: boolean
+        description: Indicates whether or not the customer would like to receive email updates from the shop. Valid values are `true` and `false`.
+      created_at: 
+        type: string
+        format: date-time
+        description: The date and time when the customer record was created.
+      email:
+        type: string
+        description: The customer's email address.
+      first_name:
+        type: string
+        description: The customer's first name.
+      order_id:
+        type: integer
+        format: integer
+        description: Unique numeric identifier for the customer.
+      last_name:
+        type: string
+        description: Customer's last name.
+      note: 
+        type: string
+        description: Extra information about the customer.
+      orders_count:
+        type: integer
+        description: The number of orders placed by this customer to a shop.
+      state:
+        type: string
+        description: Description.
+      total_spent:
+        type: integer
+        description: Total amount spent by the customer.
+      updated_at: 
+        type: string
+        format: date-time
+        description: The date and time when the customer record was last updated.
+      tags:
+        type: string
+        description: Tags are additional short descriptors, commonly used for filtering and searching, formatted as a string of comma-separated values. For example, if an order has three tags - `tag1`, `tag2`, `tag3`.
+
+  Error:
+    type: object
+    description: Object describing errors (values) on specific attributes (keys) that prevented a draft order from being saved.
+    additionalProperties:
+      description: Single `key:value` pair out of possible multiple retruned to describe errors on specific attributes.
+      properties:
+        key:
+          type: string
+          description: Attribute that raised the error.
+        value: 
+          type: string
+          description: Error description.
+            
```

Comment:
Typo on "returned"

I'd like to get others' opinions (@dominiquesr) on the definition of the `Error` object. To me this still kind of reads that the object I'd be getting back would look like:

  `{ key: <field name>, value: <error> }`

  since `key` and `value` are defined as **properties** of the error object (or rather, properties of the additionalProperties), although I know we've already chatted about this and we're not sure there's a better way...